expectMessage/utils/colorizeLog.ts

```diff
@@ -46,12 +47,42 @@ const createStyle = (cat: any, styleName?: string): string => {
   return styleName ? `\x1b[${styles[cat][styleName.toLowerCase()]}m` : "";
 };
 
-export const colorize = (string: string, customStyle: TerminalCustomStyle = {}): string => {
+const createDefault = (
+  string: string,
+  fg: string,
+  bg: string,
+  effects = ""
```

Comment:
```suggestion
```
`effects` can be `undefined` so no need to initialise it.

Reply:
It's more to give it a default value so that it doesn't have to be referenced in the arguments everytime!